backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -137,4 +137,21 @@
         <tagDatabase tag="db_version_2.4"/>
     </changeSet>
 
+    <changeSet id="add-workspace-id-to-project" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="project" columnName="workspace_id"/>
+            </not>
+        </preConditions>
+        <addColumn tableName="project">
+            <column name="workspace_id" type="int8"/>
```

Comment:
How big is int8?

Reply:
`int8` is a 64-bit value
https://github.com/liquibase/liquibase/blob/master/liquibase-core/src/main/java/liquibase/datatype/core/BigIntType.java